docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -981,7 +980,214 @@ capacity: Annotated[int, PiggybackShadowVariable(shadow_variable_name="transport
 ----
 ====
 
-==== Shadow variable cloning
+[#declarativeShadowVariable]
+=== Declarative Shadow Variable
+
+Declarative shadow variables are the simplest way to calculate shadow variables.
+This approach allows access to all the fields and methods of the declaring class.
+
+[WARNING]
+====
+This feature is currently in xref:/upgrading-timefold-solver/backwards-compatibility.adoc#previewFeatures[preview] and may be subject to change.
+====
+
+To define a declarative shadow variable: Annotate the field with `@ShadowVariable` and specifying the name of a supplier method.
```

Comment:
```suggestion
To define a declarative shadow variable: Annotate the field with `@ShadowVariable` and specify the name of a supplier method.
```

Seems like gooder English.
Another alternative would be "Annotate..., specifying...".
It's just the "and" that doesn't fit. But I'm not a native speaker, so...